server/logger.py

```diff
@@ -0,0 +1,26 @@
+import time
+import logging
+
+def Logger(name):
```

Comment:
This method should create and attach the appropriate handlers and formatters to the *root* log. Moreover, it should verify that this only gets done once, as we don't want many identical handlers attached to the same log. Then return a logger for `name`.

The goal is to have every different subsystem writing to a different sub-log, but having them all aggregated and written by the root log.